ncm-filesystems/src/main/pan/components/filesystems/schema.pan

```diff
@@ -8,10 +8,14 @@ include 'quattor/schema';
 include 'quattor/blockdevices';
 include 'quattor/filesystems';
 
+@documentation{
+when manage_blockdevs is false, filesystems does same as fstab
```

Comment:
I don't like the alias idea - that makes it the same code that can destroy data and is only protected by the name through which the code is invoked. ncm-filesystems should be a separate component (a subclass to avoid code duplication) and people that want to use it should have to disable ncm-fstab to prevent it running twice (we can enforce that with templates).